### ISSUE TITLE
TypeError: str.replace is not a function

### ISSUE THREAD

**Issue by jacargentina** [mpneuried/html-extractor#3]
Updated to use latest fixes to v0.2.1 and not i'm getting this:

```
/home/user/javascript/website/node_modules/html-extractor/lib/html_extractor.js:51
      str = str.replace(this._trimRegex, "");
                ^

TypeError: str.replace is not a function
    at HTMLExtractor.module.exports.HTMLExtractor._trim (/home/user/javascript/website/node_modules/html-extractor/lib/html_extractor.js:51:17)
    at HTMLExtractor._trim (/home/user/javascript/website/node_modules/html-extractor/lib/html_extractor.js:3:59)
    at /home/user/javascript/website/node_modules/html-extractor/lib/html_extractor.js:111:31
    at Object.htmlparser.Parser.onend (/home/user/javascript/website/node_modules/html-extractor/lib/html_extractor.js:248:13)
    at Parser.onend (/home/user/javascript/website/node_modules/html-extractor/node_modules/htmlparser2/lib/Parser.js:310:32)
    at Tokenizer._finish (/home/user/javascript/website/node_modules/html-extractor/node_modules/htmlparser2/lib/Tokenizer.js:838:12)
    at Tokenizer.end (/home/user/javascript/website/node_modules/html-extractor/node_modules/htmlparser2/lib/Tokenizer.js:829:25)
    at Parser.end (/home/user/javascript/website/node_modules/html-extractor/node_modules/htmlparser2/lib/Parser.js:338:18)
    at HTMLExtractor.module.exports.HTMLExtractor._extract (/home/user/javascript/website/node_modules/html-extractor/lib/html_extractor.js:256:14)
    at HTMLExtractor._extract (/home/user/javascript/website/node_modules/html-extractor/lib/html_extractor.js:3:59)
    at HTMLExtractor.module.exports.HTMLExtractor.extract (/home/user/javascript/website/node_modules/html-extractor/lib/html_extractor.js:94:12)
    at HTMLExtractor.extract (/home/user/javascript/website/node_modules/html-extractor/lib/html_extractor.js:3:59)
    at site_crawler.on (/home/user/javascript/website/search.js:212:22)
    at emitThree (events.js:116:13)
    at emit (events.js:194:7)
    at emitFetchComplete (/home/user/javascript/website/node_modules/simplecrawler/lib/crawler.js:1065:17)
App stopped unexpectedly
```

**Comment by jacargentina** [comment]
@mpneuried already working on the fix. A PR will land soon. It is related to the `list: true`  when using reduce

**Comment by mpneuried** [comment]
Thanks for the hint and pull request with the trimming in combination with the list feature.
I optimized your push and published it as 0.2.2
